service/history/config/config.go

```diff
@@ -379,6 +380,16 @@ func New(dc *dynamicconfig.Collection, numberOfShards int, storeType string, isA
 	return cfg
 }
 
+// NewForTest create new history service config for test
+func NewForTest() *Config {
```

Comment:
Maybe rename to NewTestConfig?

Reply:
I didn't include the word config in the function name because the package name is already config, so caller will write something like `config.NewForTest()`. Changing to `config.NewTestConfig()` sounds redundant to me.